avm/res/compute/virtual-machine/main.bicep

```diff
@@ -301,6 +340,10 @@ var windowsConfiguration = {
     ? {
         patchMode: patchMode
         assessmentMode: patchAssessmentMode
+        automaticByPlatformSettings: {
```

Comment:
Is this relevant for linux too? https://learn.microsoft.com/en-us/azure/templates/microsoft.compute/virtualmachines?pivots=deployment-language-terraform#linuxvmguestpatchautomaticbyplatformsettings-2

Reply:
Done